apps/fabric-website/src/pages/HomePage/HomePage.styles.ts

```diff
@@ -0,0 +1,421 @@
+import { IStyle, getGlobalClassNames } from 'office-ui-fabric-react';
+import { MotionDurations, MotionTimings, FontSizes } from '@uifabric/fluent-theme';
+import { IHomePageStyleProps, IHomePageStyles } from './HomePage.types';
+import { appPadding, mediaQuery } from '../../styles/constants';
+
+const CONTENT_WIDTH = 1536;
+
+const GlobalClassNames: { [key in keyof IHomePageStyles]: string } = {
+  root: 'ms-HomePage',
+  heroSection: 'ms-HomePage-heroSection',
+  heroTitle: 'ms-HomePage-heroTitle',
+  platformCardsSection: 'ms-HomePage-platformCardsSection',
+  platformsSection: 'ms-HomePage-platformsSection',
+  platformsTitle: 'ms-HomePage-platformsTitle',
+  resourcesSection: 'ms-HomePage-resourcesSection',
+  resourcesTitle: 'ms-HomePage-resourcesTitle',
+  usageSection: 'ms-HomePage-usageSection',
+  usageTitle: 'ms-HomePage-usageTitle',
+  usageIconList: 'ms-HomePage-usageIconList',
+  usageIconListItem: 'ms-HomePage-usageIconListItem',
+  usageIcon: 'ms-HomePage-usageIcon',
+  sectionContent: 'ms-HomePage-sectionContent',
+  oneHalf: 'ms-HomePage-oneHalf',
+  oneFourth: 'ms-HomePage-oneFourth',
+  inner: 'ms-HomePage-inner',
+  card: 'ms-HomePage-card',
+  cardTitle: 'ms-HomePage-cardTitle',
+  cardList: 'ms-HomePage-cardList',
+  cardListItem: 'ms-HomePage-cardListItem',
+  cardIcon: 'ms-HomePage-cardIcon',
+  link: 'ms-HomePage-link',
+  linkIcon: 'ms-HomePage-linkIcon',
+  linkText: 'ms-HomePage-linkText',
+  illustration: 'ms-HomePage-illustration'
+};
+
+export const getStyles = (props: IHomePageStyleProps): IHomePageStyles => {
+  const { theme, className, isMountedOffset, isInverted, beforeColor, afterColor } = props;
+  const { palette } = theme;
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  const sectionAnimation: IStyle = [
+    {
+      opacity: 0,
+      transform: 'translate3d(0, 48px, 0)',
+      transition: `
+        transform ${MotionDurations.duration3} ${MotionTimings.decelerate},
```

Comment:
Would be better in this case to put two strings on separate lines and concatenate them. Otherwise you'll end up with tons of unnecessary white space in the output.